build.js

```diff
@@ -396,6 +398,26 @@ function handle(options) {
   }
 }
 
+function wrapWithScript(assertions) {
+  // wrap a single test in an array so we can deal with them consistently
+  if (assertions) {
+    assertions = [].concat(assertions);
+  }
+
+  return assertions.filter(function(assertion) {
+    return assertion && assertion.script;
+  })
+  .map(function(assertion) {
+    var $ = cheerio.load(''
+        + '<script' + (assertion.type ? ' type="' + assertion.type + '"' : '') + '>'
+        + assertion.script
+        + '\n</script>'
```

Comment:
`\n` added here, but not after `<script>`. It's inconsistency. It should be in both places or nowhere. The rest LGTM.

Reply:
It’s needed here for single-line comments; it’s not needed after the closing tag. So no matter what, it can’t be nowhere. I’m in favor of including it after as well, tho.

Reply:
I mean opening tag. At this moment, it's nowhere and it's the reason of changes in generated HTML.

Reply:
I only see one before the closing tag - but sure, fair enough.